### PR TITLE
#91 & # 108 add fetch mail field and css changed

### PR DESCRIPTION
**Pull request by pravinpimpale**

### CODE REVIEW

**Comment by pravinpimpale** [comment]
It is done
Screenshot (115)

File: public/css/bootstrap.css
Comment (Shubham40Bears) [inline public/css/bootstrap.css:1]: Do not change this file

File: resources/views/mailboxes/sidebar_menu_view.blade.php [hunk 1]
```diff
@@ -21,6 +21,7 @@
             </div>
         @endif
         <a class="btn btn-trans" href="{{ route('conversations.create', ['mailbox_id' => $mailbox->id]) }}" aria-label="{{ __("New Conversations") }}" data-toggle="tooltip" title="{{ __("New Tickets") }}" role="button"><i class="glyphicon glyphicon-envelope"></i></a>
+        <a class="btn btn-trans" href="{{route('mailboxes.fetchMail')}}" data-toggle="tooltip" title="{{ __("Fetch Tickets") }}" role="button" value="fetch_emails" type="submit" name="action"><i class="glyphicon glyphicon-refresh"></i></a>
```
Comment (Shubham40Bears) [inline resources/views/mailboxes/sidebar_menu_view.blade.php]: anchor tag cannot be submit

**Review by Shubham40Bears — CHANGES_REQUESTED** [review]

File: resources/views/mailboxes/sidebar_menu_view.blade.php (hunk 1 above)
Comment (pravinpimpale) [reply resources/views/mailboxes/sidebar_menu_view.blade.php]: removed type

**Review by Shubham40Bears — APPROVED** [review]

**Pull request merged**